api/cases/case-list.js

```diff
@@ -12,16 +12,17 @@ function getOnlineHearing(caseIds, options) {
 }
 
 function rawCasesReducer(cases, columns) {
-    return cases.map(caseRow => {
-        return {
-            case_id: caseRow.id,
-            case_reference: valueProcessor(sscsCaseListTemplate.case_number.label, caseRow),
-            case_fields : columns.reduce((row, column) => {
-                row[column.case_field_id] = valueProcessor(column.value, caseRow);
-                return row;
-            }, {})
-        };
-    });
+    return cases.filter(row => (row.case_data.caseReference != null && row.case_data.caseReference != undefined))
```

Comment:
This filtering needs to happen AFTER the transpilation to the template as caseReference is specific to SSCS data structures and will break for any other jurisdictions. i.e. filtering needs to be done on `case_number.label` instead

Reply:
Moved the filtering code to after template is applied.